packages/app/cypress/e2e/create-spec.cy.ts

```diff
@@ -0,0 +1,40 @@
+import type { e2eProjectDirs } from '@packages/frontend-shared/cypress/e2e/support/e2eProjectDirs'
+
+function scaffoldAndVisit(project: typeof e2eProjectDirs[number], type: 'component' | 'e2e') {
+  cy.scaffoldProject(project)
+  cy.openProject(project)
+  cy.startAppServer(type)
+  cy.__incorrectlyVisitAppWithIntercept()
```

Comment:
Any reason to use `__incorrectlyVisitAppWithIntercept` vs `visitApp`?

Reply:
I have no idea what this does differently - I just copied pasted this from the other tests that do the same thing. I'd like to find at some point, but for now it seems like this is how tests in `app` look. Maybe we should create a custom command at some point.